pkg/virt-launcher/virtwrap/converter/network.go

```diff
@@ -81,23 +91,9 @@ func CreateDomainInterfaces(vmi *v1.VirtualMachineInstance, domain *api.Domain,
 			domainIface.ACPI = &api.ACPI{Index: uint(iface.ACPIIndex)}
 		}
 
-		if iface.Bridge != nil || iface.Masquerade != nil {
-			// TODO:(ihar) consider abstracting interface type conversion /
-			// detection into drivers
-
+		if tapDomainAttachment {
```

Comment:
By using this bool instead the condition is true also when `iface.Binding != nil`, right?